pages/home/arcade-sdks.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Arcade SDKs"
+description: "Arcade SDKs"
+---
+
+# Arcade SDKs
+
+Arcade provides SDKs for several languages.  These SDKs make it easy to use Arcade's tools within your own agents and applications.
+
+## Python SDK
+
+Install with: `pip install arcadepy`
+
+Learn more about the [Python SDK](https://github.com/ArcadeAI/arcade-py).
+
+## JavaScript / TypeScript SDK
+
+Install with: `npm install @arcadeai/arcadejs`
+
+Learn more about the [JavaScript / TypeScript SDK](https://github.com/ArcadeAI/arcade-js).
+
+## Go SDK
+
+Install with: `go get -u 'github.com/stainless-sdks/arcade-engine-go'`
```

Comment:
Why is this `arcade-engine`? Is it only for the engine APIs?

Reply:
Should we fix stainless to be consistent with the other SDKs?

Reply:
yeah, and it's not pointing to our public github either... digging